depmap/nbconverted/0.download_depmap.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Download Dependency Data
+# 
+# Source: [Cancer Dependency Map resource](https://depmap.org/portal/download/).
+# 
+# - CRISPRGeneDependency.csv: The data in this document describes the probability that a gene knockdown has an effect on cell-inhibition or death. These probabilities are derived from the data contained in CRISPRGeneEffect.csv using methods described [here](https://www.biorxiv.org/content/10.1101/720243v1).
```

Comment:
Are there column datatypes or more formal schema definition provided with these files? Here I ask out of thinking about the potential for datatypes to be potentially misread from CSV's.

Reply:
yes, this is good to keep in mind. But i do not believe there is such a schema